lib/Utils/Architectures.pm

```diff
@@ -195,4 +196,16 @@ sub is_zvm {
     return (get_var('MACHINE') =~ /zvm/i);
 }
 
+=head2 is_32bit
+
+ is_32bit();
+
+Returns C<true if machine is 32 bit architecture>.
+
+=cut
+
+sub is_32bit {
+    return is_i586 || is_i686 || is_arm;
```

Comment:
arm is 32bit?

Reply:
Thank you for your reply (can wait till January).

Yes 'arm' is 32 bit: https://openqa.opensuse.org/tests/2901418#settings test with `ARCH=arm` also mentions `armv7` (in 
https://openqa.opensuse.org/assets/repo/openSUSE-Leap-15.3-oss-armv7hl-Build10.754) which is 32 bit (https://en.wikipedia.org/wiki/ARM_architecture_family#Cores) and also see `opensuse-15.3-arm-10.754-textmode@aarch32.qcow2` which mentions `aarch32`. 64 bit is called either `aarch64` or `arm64` (in kernel sources).

Reply:
NOTE: IMHO we don't have 32 bit powerpc (there is no `is_ppc`), nor 32 bit s390 (there is no `is_s390`), thus these are left.